src/WebJobs.Script/Host/IScriptHostFactory.cs

```diff
@@ -13,6 +13,6 @@ public interface IScriptHostFactory
             IScriptEventManager eventManager,
             ScriptSettingsManager settingsManager,
             ScriptHostConfiguration config,
-            ILoggerFactoryBuilder loggerFactoryBuilder);
+            ILoggerProviderFactory loggerFactoryBuilder);
```

Comment:
nit: rename param to match type rename?

Reply:
Yes, I've done that in my current changes (to be pushed soon)